### ISSUE TITLE
Unexpected drop of otel metrics

### ISSUE THREAD

**Issue by chenlujjj** [VictoriaMetrics/VictoriaMetrics#6343]
### Is your question request related to a specific component?

VictoriaMerics cluster - vminsert

### Describe the question in detail

Hi Team,

We are using otel collector to push metrics to Victoria Metrics cluster via otlphttp exporter, it works great generally.

We found some warning logs of vminsert component:
```
{"ts":"2024-05-24T08:28:14.618Z","level":"warn","caller":"VictoriaMetrics/lib/protoparser/opentelemetry/stream/streamparser.go:88","msg":"unsupported type for metric \"http.client.duration\""}
```

The `http.client.duration` metric should be of histogram type according to the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/metrics/semantic_conventions/http-metrics.md#metric-httpclientduration). And from the [code](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/cluster/lib/protoparser/opentelemetry/stream/streamparser.go#L87), the histogram metrics should not be dropped.

Could you please take a look at it? Thanks in advance!

### Troubleshooting docs

- [X] General - https://docs.victoriametrics.com/troubleshooting/
- [ ] vmagent - https://docs.victoriametrics.com/vmagent/#troubleshooting
- [ ] vmalert - https://docs.victoriametrics.com/vmalert/#troubleshooting

**Comment by zekker6** [comment]
@chenlujjj Could you please share a config which you're using for otlphttp? (without any sensitive details)
It will make it easier to reproduce the issue.

**Comment by chenlujjj** [comment]
@zekker6 Sure, below is part of the otel-collector configuration related to the otlphttp
```
exporters:
  otlphttp:
        metrics_endpoint: https://<vminsert-host>/insert/0/opentelemetry/api/v1/push
service:
  pipelines:
    metrics/application_infra_metric_center:
          exporters:
          - otlphttp
          processors:
          - memory_limiter
          - k8sattributes
          - batch
          - resourcedetection
          - resource
          receivers:
          - signalfx
          - otlp
```

**Comment by chenlujjj** [comment]
Thanks @AndrewChubatiuk 
Is it possible to log the field names as well ?

**Comment by AndrewChubatiuk** [comment]
added exponential histograms support in this MR instead

**Comment by chenlujjj** [comment]
hi @AndrewChubatiuk just wondering when we can expect https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6354 to be merged ?

**Comment by Shindek77** [comment]
hi @AndrewChubatiuk , when we can expect above mentioned PR to be merged ? We want send the ExponentialHistogram Metrics on VM

**Comment by f41gh7** [comment]
The feature was included into [v1.105.0](https://docs.victoriametrics.com/changelog/#v11050)
